src/main/java/webserver/request/Path.java

```diff
@@ -0,0 +1,31 @@
+package webserver.request;
+
+import java.util.Map;
+
+import utils.ParseUtils;
+
+public class Path {
+
+    private final String path;
+    private Map<String, String> params;
+
+    public Path(String path) {
+        this.path = path;
+        if (path.contains("?")) {
+            parseParams();
+        }
+    }
+
+    private void parseParams() {
+        String[] tokens = path.split("\\?");
+        params = ParseUtils.parseParam(tokens[1]);
+    }
+
+    public String getPathWithoutParam() {
+        return path.split("\\?")[0];
+    }
+
+    public Map<String, String> getParams() {
+        return params;
```

Comment:
```suggestion
        return new HashMap<>(params);
```
별건 아니지만 방어적 복사를 해주는 것도 의미가 있을 것 같네요 :)